client/src/pages/Unauthorized/useStyles.ts

```diff
@@ -0,0 +1,21 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    minHeight: '100vh',
+    '& .MuiInput-underline:before': {
+      borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
+    },
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  boxWrapper: {
+    display: 'flex',
```

Comment:
Most of these styles can be applied to the Box component directly

Eg: `<Box display="flex" alignItems="center"...>`